### ISSUE TITLE
Unable to run python cells due to ImportError

### ISSUE THREAD

**Issue by jordan-schneider** [polynote/polynote#541]
I'm unable to run python cells due to the following error:

```
    Uncaught exception: <class 'ImportError'>: /usr/local/lib/python3.8/lib-dynload/_struct.cpython-38-x86_64-linux-gnu.so: undefined symbol: PyFloat_Type (jep.JepException)

        /usr/local/lib/python3.8/struct.<module>(struct.py:13)
        /usr/local/lib/python3.8/pickle.<module>(pickle.py:33)
        /home/$USER/.local/lib/python3.8/site-packages/parso/cache.<module>(cache.py:14)
        /home/$USER/.local/lib/python3.8/site-packages/parso/grammar.<module>(grammar.py:10)
        /home/$USER/.local/lib/python3.8/site-packages/parso/__init__.<module>(__init__.py:42)
        /home/$USER/.local/lib/python3.8/site-packages/jedi/api/__init__.<module>(__init__.py:16)
        /home/$USER/.local/lib/python3.8/site-packages/jedi/__init__.<module>(__init__.py:38)
        <string>.<module>(<string>:2)
        jep.Jep.exec(Jep.java:-2)
        jep.Jep.exec(Jep.java:478)
        polynote.kernel.interpreter.python.PythonInterpreter$$anonfun$exec$1.apply(PythonInterpreter.scala:182)
        polynote.kernel.interpreter.python.PythonInterpreter$$anonfun$exec$1.apply(PythonInterpreter.scala:182)
        polynote.kernel.interpreter.python.PythonInterpreter$$anonfun$jep$1.apply(PythonInterpreter.scala:181)
        zio.blocking.Blocking$Service$$anonfun$effectBlocking$1$$anonfun$apply$3.apply(Blocking.scala:133)
        zio.blocking.Blocking$Service$$anonfun$effectBlocking$1$$anonfun$apply$3.apply(Blocking.scala:127)
        zio.internal.FiberContext.evaluateNow(FiberContext.scala:333)
        zio.internal.FiberContext.zio$internal$FiberContext$$run$body$2(FiberContext.scala:602)
        zio.internal.FiberContext$$anonfun$7.run(FiberContext.scala:602)
        java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        java.lang.Thread.run(Thread.java:748)
```

I no nothing about jep, so I apologize if this is a common error. Java 8, Ubuntu 18.04.

**Comment by jonathanindig** [comment]
Hi @skinnersBoxy, welcome to Polynote and thanks for making an issue! 

Can you provide us with a little more information, such as the contents of the notebook and cell you were trying to run, dependencies added to the notebook (if any), and the version of Python you're using? Thanks!

**Comment by skinnersBoxy** [comment]
The notebook has two cells, one with `a="str"`, the other with `a=5`. There are no dependencies, and I'm using python 3.8. Is 3.8 not supported?

**Comment by jonathanindig** [comment]
Looks like this is related to https://bugs.python.org/issue35894

**Comment by jonathanindig** [comment]
Resolved by #551 

Please open a new issue if you are still running into problems. Note that python 3.8 is not supported by Spark until version 3+
